src/app.js

```diff
@@ -13,6 +13,10 @@ app.use(express.json())
 
 app.use('/v1', routes)
 
+app.use((err, req, res, next) => {
+  res.status(400).send({ error: err.message })
```

Comment:
```suggestion
  res.status(500).send({ error: err.message })
```
Se o erro não foi tratado por nós em nenhum lugar antes de chegar aqui, acredito que não seja um erro genérico de mal uso.

Reply:
Ele cai nessa função por exemplo quando a pessoa esquece de mandar o token na req
Vou deixar 500, mas vou ver se tem um jeito de setar um código específico pra cada throw dentro da aplicação